static/styles/_components/_filters.scss

```diff
@@ -76,6 +82,17 @@
   }
 }
 
+.filter__header {
+  padding: 0 1rem;
+}
+
+.accordion__item {
```

Comment:
Good catch. Moved to `_side-panel.scss` along with other modifications to the accordion components from being within `.side-panel`.